general/server/users/adding-managing-users.md

```diff
@@ -36,7 +36,10 @@ For the following options it should be noted that if you never set up Live TV, u
 #### Media Playback
 `Allow media playback` Unchecking this option will block the user's access to media libraries, this does not include Live TV.
 
-`Allow audio/video playback that requires transcoding` Unchecking this option will block the user's access to video playback that requires transcoding (Decreases bandwidth use and quality)
+> [!NOTE]
+> More information about transcoding can be found on it's own page - [Transcoding](https://jellyfin.org/docs/general/server/transcoding.html)

```

Comment:
cannot we make a relative link that doc compiler would turn into a full one? cc @EraYaN